### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by heikkiquixey**
@mquander @addumb Please review.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -31,12 +31,12 @@
       version=aliyun_version,
       description='Python client for Aliyun APIs.',
       author='Adam Gray, Akshay Dayal',
-      author_email='adam@addumb.com, akshay@quixey.com',
```
Comment (addumb) [inline setup.py]: Add yourself?

**Comment by addumb** [comment]
Looks good! I think there's something up in find_credentials:

```
python ./ali.py -S ecs -r cn-hangzhou Action=DescribeRegions
Traceback (most recent call last):
  File "./ali.py", line 85, in <module>
    secret_access_key=args.secret_access_key)
  File "/home/adam/qsrc/python-aliyun/aliyun/connection.py", line 119, in __init__
    creds = find_credentials()
  File "/home/adam/qsrc/python-aliyun/aliyun/connection.py", line 79, in find_credentials
    creds.access_key_id = cp.get('default', 'access_key_id')
AttributeError: can't set attribute
```

Ah yeah, namedtuples are immutable, and the config file discovery mechanism isn't working.

**Comment by heikkiquixey** [comment]
Wait, this code never worked?

**Comment by addumb** [comment]
Whoaaaaaaaaaa we were actually adding attributes to the namedtuple module not to the named tuple itself... and passing those around as if they were credentials.

**Pull request merged**